### DISCUSSION TITLE
Problem using Relation Reference in attribute form.

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5208] [Q&A]
Hi everyone,
I am working on a QGIS project to upload on Qfield, and in this project, I have created a 1:n relation between a table with a point geometry that represents trees and another table without geometry that represents the conditions of the trees. But when I choose the "Relation Reference" widget type in the attribute form of the main table, the "Relation" tab is visible. However, the "Relation" dropdown is empty, which means that the project hasn't recognized the relation I have created. Is there anyway to resolve this?

**Comment by oolonek** [comment]
Did you set up the Relations at the project level ?
See https://gis.stackexchange.com/questions/403755/qgis-relation-reference-display-expression
